src/app/layouts/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -20,8 +20,27 @@ export const NavItem = ({ label, href, dropdownInner }: NavItemProps) => {
   const onClose = () => setDropdownOpen(false);
   const onOpen = () => setDropdownOpen(true);
 
+  const location = useLocation();
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      setDropdownOpen((prev) => !prev);
+    }
+  };
```

Comment:
I'd expect support for some additional keys, like in the following example: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/

Reply:
maybe we should consider to integrate aria components that is out of the box accessible e.g. Headless UI

Reply:
@Quiddlee good point, let's discuss it at our next call

cc: @ansivgit

Reply:
I'v added support for space and esc keys. Not sure about those "optional" one from example, as it will require a lot of structure change not just in this component, but in header also. Or manipulation of nodes with queryselector / tons of refs.